src/values/int_value.rs

```diff
@@ -444,6 +447,9 @@ impl IntValue {
         if !self.is_const() {
             return None;
         }
+        if self.get_type().get_bit_width() > 64 {
```

Comment:
> LLVMConstIntGet[SZ]ExtValue are convenience functions to access the value of your constant integer when it is 64 bits or less. Return None for larger integers.

LLVM supports 128bit ints as well as custom bit width ints. Why wouldn't this work for > 64?

Reply:
> > LLVMConstIntGet[SZ]ExtValue are convenience functions to access the value of your constant integer when it is 64 bits or less. Return None for larger integers.
> 
> LLVM supports 128bit ints as well as custom bit width ints. Why wouldn't this work for > 64?

On common platforms the largest type available in C and C++ is a 64-bit type, so LLVM provided a convenience function that returns a `uin64_t`. (This is akin to the fact that llvm supports 256-bit integers but rust only has a `u128`.) If the value is larger, LLVMConstIntGet[SZ]ExtValue will saturate. In C++ you can access the larger value with ConstantInt::getValue(), but there doesn't appear to be any way at all to get it through the C API.